test/kata7.getEmployerRole.test.js

```diff
@@ -1,5 +1,27 @@
 const { getEmployerRole } = require("../src");
 
 describe("getEmployerRole", () => {
-  test("returns the employee's role in the company", () => {});
+  it("returns the employee's role in the company", () => {
+    const employees = [{
+        name: "Satti",
+
+        role: "Developer",
+      },
+      {
+  
+      name: "Jenny",
+  
+      role: "Sales Associate",
+  
+  }, {
+        name: "Javid",
+
+  
+      role: "Human Recommended Reading Assistant",
+  
+  }];
```

Comment:
Formatting is a bit out here, theres a lot of extra blank lines and the indentation is a bit scatty. Should look like this:
```
    const employees = [
      {
        name: "Satti",
        role: "Developer",
      },
      {
        name: "Jenny",
        role: "Sales Associate",
      },
      {
        name: "Javid",
        role: "Human Recommended Reading Assistant",
      },
    ];
```